server/@types/dashboard/index.d.ts

```diff
@@ -1,13 +1,6 @@
 export type MigrationViewFilter = {
+  prisonId?: string
```

Comment:
Are these changes for the filtering of previous migrations?  I thought we'd decided that we were not going to do that view filter for any other projects besides visits as it was wasn't really necessary

Reply:
Did we? ... I just thought that as appointments will be migrated on a per-prison basis, that this would be useful?

Reply:
it wasn't in the end for visits, so we had a chat a while ago and said not to bother.